packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeComponent.tsx

```diff
@@ -222,12 +232,17 @@ function View2DButton(props: ModelsTreeHeaderButtonProps) {
   return (
     <IconButton
       className="tree-widget-header-tree-toolbar-icon"
-      icon={is2dToggleActive ? "icon-visibility" : "icon-visibility-hide-2"}
+      size="small"
+      styleType="borderless"
       title={TreeWidget.translate("toggle2DViews")}
       onClick={viewToggle2D}
-      label={TreeWidget.translate("label2D")}
       disabled={models2d.length === 0}
-    />
+    >
+      <div className="tree-widget-header-tree-toolbar-icon-label">
```

Comment:
Looks like these buttons can be changed to https://itwin.github.io/iTwinUI/react/?path=/story/buttons-button--with-icon